schemas/clinvar-study.tables.json

```diff
@@ -0,0 +1,3068 @@
+          },
+          {
+            "name": "taxonomy_id",
+            "datatype": "integer"
+          }
+        ]
+      },
+      {
+        "name": "interpreted_record",
+        "columns": [
+          {
+            "name": "persistent_id",
+            "datatype": "string"
+          },
+          {
+            "name": "simple_allele",
+            "datatype": "string"
+          },
+          {
+            "name": "haplotype",
+            "datatype": "string"
+          },
+          {
+            "name": "geno_type",
+            "datatype": "string"
+          },
+          {
+            "name": "review_status",
+            "datatype": "string"
+          },
+          {
+            "name": "rcv_list",
+            "datatype": "string"
+          },
+          {
+            "name": "clinical_assertion_list",
+            "datatype": "string"
+          },
+          {
+            "name": "trait_mapping_list",
+            "datatype": "string"
+          },
+          {
+            "name": "deleted_scv_list",
+            "datatype": "string"
+          },
+          {
+            "name": "general_citations",
+            "datatype": "string",
+            "array_of": true
+          }
+        ]
+      },
+      {
+        "name": "type_allele",
+        "columns": [
+          {
+            "name": "persistent_id",
+            "datatype": "string"
+          },
+          {
+            "name": "gene_list",
+            "datatype": "string"
+          },
+          {
+            "name": "name",
+            "datatype": "string"
+          },
+          {
+            "name": "variant_type",
+            "datatype": "string"
+          },
+          {
+            "name": "location",
+            "datatype": "string"
+          },
+          {
+            "name": "other_name_list",
+            "datatype": "string"
+          },
+          {
+            "name": "protein_change",
+            "datatype": "string",
+            "array_of": true
+          },
+          {
+            "name": "hgvs_list",
+            "datatype": "string"
+          },
+          {
+            "name": "interpretations",
+            "datatype": "string"
+          },
+          {
+            "name": "xref_list",
+            "datatype": "string"
+          },
+          {
+            "name": "comment",
+            "datatype": "string"
+          },
+          {
+            "name": "functional_consequence",
+            "datatype": "string"
+          },
+          {
+            "name": "allele_frequency_list",
+            "datatype": "string"
+          },
+          {
+            "name": "global_minor_allele_frequency",
+            "datatype": "string"
+          },
+          {
+            "name": "allele_id",
+            "datatype": "integer"
+          },
+          {
+            "name": "variation_id",
+            "datatype": "integer"
+          }
+        ]
+      },
+      {
+        "name": "gene_list",
```

Comment:
What do you think about removing this table? Wherever there are references to it now, we could have a column:
```json
{ "name": "gene_ids", "datatype": "string", "array_of": true }
```
BigQuery has a length function on array columns that can be used to rebuild the gene_count value.